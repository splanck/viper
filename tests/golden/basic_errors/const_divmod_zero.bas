10 PRINT 1 \ 0
20 PRINT 1 MOD 0
30 END
