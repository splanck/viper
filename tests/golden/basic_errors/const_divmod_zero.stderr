B2002.*B2002
